### PR TITLE
secrethub: add page

### PR DESCRIPTION
**Pull request by SimonBarendse**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/secrethub.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`secrethub generate {{path/to/secret}}`
+
+- Store a value from the clipboard as a new secret (version):
+
+`secrethub write --clip {{path/to/secret}}`
+
+- Store a value supplied on stdin as a new secret (version):
+
+`echo "secret value" | secrethub write {{path/to/secret}}`
```
Comment (sbrl) [inline pages/common/secrethub.md]: Perhaps we could put `secret value` in the token syntax here please?
```suggestion
`echo "{{secret value}}" | secrethub write {{path/to/secret}}`
```

Comment (zdroid) [reply pages/common/secrethub.md]:
```suggestion
`echo "{{secret_value}}" | secrethub write {{path/to/secret}}`
```

Should be with an underscore.

Comment (sbrl) [reply pages/common/secrethub.md]: Updated my suggestion :+1:

Comment (SimonBarendse) [reply pages/common/secrethub.md]: Thanks! I've updated the example :+1:

**Comment by SimonBarendse** [comment]
Thanks for being on top of this! :rocket:

File: pages/common/secrethub.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# secrethub
+
+> A tool to keep secrets out of config files.
+> More information: <https://secrethub.io>.
+
+- Print a secret to stdout:
+
+`secrethub read {{path/to/secret}}`
+
+- Generate a random value and store it as a new secret (version):
```
Comment (agnivade) [inline pages/common/secrethub.md]: What does the word (version) in the parentheses mean ? It's not very clear to me.

Comment (SimonBarendse) [reply pages/common/secrethub.md]: Hi @agnivade, great feedback! :smile:

When writing to a path where a secret is already stored, a new version of that secret is created.

When reading a secret, it's possible to read a specific version with: `{{path/to/secret}}:{{version}}`, e.g. `path/to/secret:2`. When the version is omitted (as in the first example of this tldr page), it defaults to `path/to/secret:latest`, which retrieves the latest version of the secret.

It's a challenge to keep it concise while also explaining it enough :smile: I've opened https://github.com/tldr-pages/tldr/pull/3847 as an attempt to make this clearer. What do you think? More feedback is welcome :+1:

Comment (agnivade) [reply pages/common/secrethub.md]: Yes, the new PR looks good to me. Thanks !

**Pull request merged**